### ISSUE TITLE
Expose chai's assert interface

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#614]
### Is this a Feature or Bug?
Feature

### Current behavior:
Chai's assert does not work.

```js
assert.equal(1, 1, "one is one")
assert.isOk({}, "is okay")
assert.isFalse(false, "is false")
```
![screen shot 2017-08-31 at 10 46 13 pm](https://user-images.githubusercontent.com/1268976/29953508-e46b0a94-8e9e-11e7-89c3-58f0c3b5418b.png)

### Desired behavior:
We expose chai's `expect` interface, and should also expose `assert`.

**Comment by brian-mann** [comment]
![screen shot 2017-08-31 at 10 46 58 pm](https://user-images.githubusercontent.com/1268976/29953518-005b9656-8e9f-11e7-8e9d-ccda53c32ef7.png)

The code for this is done, but has yet to be released. We'll update this issue and reference the changelog when it's released.

**Comment by jennifer-shehane** [comment]
Duplicate of #101

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
